### ISSUE TITLE
Add content of all plenary sessions

### ISSUE THREAD

**Issue by raosudha89** [acl-org/acl-2020-virtual-conference#205]
- [ ] Add slideslive id for all plenary sessions

- [ ] Add rocketchat ID for all QA sessions of plenary

- [ ] Add zoom link for livestream 

- [ ] Replace 'speaker/institution' with 'presenter' for non-keynote items in plenary session.

**Comment by raosudha89** [comment]
From previous issue:

- [ ]  Rename the "Speakers" to "Plenary Sessions" in the page title
- [ ]  Order the sessions by time (I didn't do the PST / GMT conversion)
- [ ]  Gather the necessary information to be displayed. Can you make a list? I'll send an email to our program chairs and cc you to get those information.

Also, I'm thinking instead of opening a new page for each plenary session, can we make those as a collapse cards? (https://getbootstrap.com/docs/4.3/components/collapse/) In this way, the audience will stay on the same page for the whole plenary sessions on this day. (Just like a physical room experience).
But let's first get the basic version working and come back to this when we have time, unless you already know an easy way to do it.
Feel free to create another ticket if you feel you need some additional volunteers to work on this together.

**Comment by georgepar** [comment]
hi @raosudha89.
do you have an excel with this information?

**Comment by hao-fang** [comment]
@georgepar Can you first put some dummy string for the SlidesLive ID, RocketChat channel name, and Zoom links?
We will need to add those once we gather all the information.

**Comment by raosudha89** [comment]
@georgepar  Since this is high-priority and needs to be done before the dry-run on Friday, I want to also work on this. Can you tell me what exactly are you working on and I can maybe work parallely on other things?

**Comment by raosudha89** [comment]
I didn't realize that @georgepar  is in a different timezone. Apologies for rushing you on this. 

I have created a PR https://github.com/acl-org/acl-2020-virtual-conference/pull/209 for adding slideslive link and rocketchat channel to plenary sessions.

I tried to use collapse cards instead of opening the plenary information in the new page. But I realized that with the bio+abstract information for keynote, the collapse card becomes too large. So I reverted back to opening up a new page for each plenary session. 

@hao-fang  let me know if that is okay.

**Comment by hao-fang** [comment]
> I didn't realize that @georgepar is in a different timezone. Apologies for rushing you on this.
> 
> I have created a PR #209 for adding slideslive link and rocketchat channel to plenary sessions.
> 
> I tried to use collapse cards instead of opening the plenary information in the new page. But I realized that with the bio+abstract information for keynote, the collapse card becomes too large. So I reverted back to opening up a new page for each plenary session.
> 
> @hao-fang let me know if that is okay.

@raosudha89 That's good -- we just want to have something working for now. Everything else can be "Website Enhancements".

**Comment by georgepar** [comment]
@raosudha89 yeah I'm on EET :) sorry for the confusion.

I will continue with this today and see what else needs to be done

**Comment by hao-fang** [comment]
@georgepar 
I have send an email to PCs and cc'd you to ask for descriptions of individual plenary sessions.

Meanwhile, can you add the speaker/host information to each session based on the table below?

Title | Authors | Type
-- | -- | --
  |   |  
Rewriting the Past: Assessing the Field through the Lens of Language Generation | Kathleen McKeown | Keynote
TBD | Josh Tenenbaum | Keynote
Test of Time Awards 2020 | Barbara Di Eugenio | Test of Time Awards
Opening Remarks | Dan Jurafsky, Joyce Chai, Natalie Schluter, Joel Tetreault | Opening Remarks
Closing Remarks | Dan Jurafsky, Joyce Chai, Natalie Schluter, Joel Tetreault | Closing Remarks
Presidential Address | Hinirich Schütze | Presidential Address
NAACL 2021 | Kristina Toutanova, Luke Zettlemoyer, Anna Rumshisky, Saif Mohammad | Conference Announcements
ACL 2021 | Chengqing Zong,  Thepchai Supnithi | Conference Announcements
EMNLP 2020 | Bonnie Webber, Trevor Cohn, Yulan He, Yang Liu | Conference Announcements
COLING 2020 | Donia Scott, Nuria Bel, Chengqing Zong | Conference Announcements
AACL 2020 | Kam-Fai Wong, Kevin Knight, Hua Wu | Conference Announcements
Lifetime Achievement Award - Speech |   | Lifetime Achievement Award
Lifetime Achievement Award (LAA) - Introduction of the recipient | Hinirich Schütze | Lifetime Achievement Award
Distinguished Service Award (DSA) - Introduction of the recipient | Hinirich Schütze | Distinguished Service Award
Distinguished Service Award (DSA) - talk by the recipient |   | Distinguished Service Award
Announcement/Introduction BP Awards | Joyce Chai, Natalie Schluter, Joel Tetreault | Best Paper Award
ACL 2023 Call for Bids | Tim Baldwin | Conference Announcements

**Comment by hao-fang** [comment]
## July 6
![image](https://user-images.githubusercontent.com/9957482/85804484-99698800-b6fe-11ea-8ce8-a434a5fdf87e.png)
## July 7
![image](https://user-images.githubusercontent.com/9957482/85804494-a38b8680-b6fe-11ea-8694-ecc2271818c0.png)
## July 8
![image](https://user-images.githubusercontent.com/9957482/85804510-abe3c180-b6fe-11ea-923a-fe74c57986c7.png)

**Comment by hao-fang** [comment]
Here is the new instruction after today's meeting.

* We will not expose Zoom links to the public now. Everyone will ask questions in RocketChat and the co-host of the Zoom meeting will pick up questions and ask the speaker (like an interview).
  - [ ] Remove `zoom_link` field from`PlenarySession` dataclass in `sitedata.py`
  - [ ] Remove `zoom_link` field from [plenary_sessions.yml](https://github.com/acl-org/acl-2020-virtual-conference-sitedata/blob/master/plenary_sessions.yml)
* Opening Remarks: This is a pre-recorded talk. No QA associated with it. No RocketChat channel.
  - [ ] Add the presentation ID to [plenary_sessions.yml](https://github.com/acl-org/acl-2020-virtual-conference-sitedata/blob/master/plenary_sessions.yml). You can find it below from the SL_PRESENTATION_ID below.
  - [ ] Update the https://virtual.acl2020.org/plenary_session_opening_remarks.html page to make sure all information are shown.
* Presidential Address







Submission ID | Title | Authors | All Author Emails | Type | SL_PRESENTATION_LINK | SL_PRESENTATION_ID
-- | -- | -- | -- | -- | -- | --
  |   |   |   |   |   |  
8000 | Rewriting the Past: Assessing the Field through the Lens of Language Generation | Kathleen McKeown | kathy@cs.columbia.edu | Keynote | slideslive.com/38929460 | 38929460
8001 | TBD | Josh Tenenbaum | jbt@mit.edu | Keynote | slideslive.com/38929461 | 38929461
8003 | Test of Time Awards 2020 | Barbara Di Eugenio | bdieugen@uic.edu | Test of Time Awards | slideslive.com/38929462 | 38929462
9000 | Opening Remarks | Dan Jurafsky, Joyce Chai, Natalie Schluter, Joel Tetreault | jurafsky@stanford.edu, chaijy@umich.edu, natschluter@itu.dk, tetreaul@gmail.com | Opening Remarks | slideslive.com/38929463 | 38929463
9001 | Closing Remarks | Dan Jurafsky, Joyce Chai, Natalie Schluter, Joel Tetreault | jurafsky@stanford.edu, chaijy@umich.edu, natschluter@itu.dk, tetreaul@gmail.com | Closing Remarks | slideslive.com/38929464 | 38929464
9002 | Presidential Address | Hinirich Schütze | hinrichacl@cis.uni-muenchen.de | Presidential Address | slideslive.com/38929465 | 38929465
9003 | NAACL 2021 | Kristina Toutanova, Luke Zettlemoyer, Anna Rumshisky, Saif Mohammad | kristinatoutanova@gmail.com, lsz@cs.washington.edu, arumshisky@gmail.com, uvgotsaif@gmail.com | Conference Announcements | slideslive.com/38929466 | 38929466
9004 | ACL 2021 | Chengqing Zong,  Thepchai Supnithi | cqzong@nlpr.ia.ac.cn, thepchai.supnithi@nectec.or.th | Conference Announcements | slideslive.com/38929467 | 38929467
9005 | EMNLP 2020 | Bonnie Webber, Trevor Cohn, Yulan He, Yang Liu | Yulan.He@warwick.ac.uk, trevor.cohn@unimelb.edu.au, yang.yangl@gmail.com, bonnie.webber@ed.ac.uk | Conference Announcements | slideslive.com/38929468 | 38929468
9006 | COLING 2020 | Donia Scott, Nuria Bel, Chengqing Zong | D.R.Scott@sussex.ac.uk, cqzong@nlpr.ia.ac.cn, nuria.bel@upf.edu | Conference Announcements | slideslive.com/38929469 | 38929469
9007 | AACL 2020 | Kam-Fai Wong, Kevin Knight, Hua Wu | aacl-ijcnlp-2020-program@googlegroups.com | Conference Announcements | slideslive.com/38929470 | 38929470
8002 | Lifetime Achievement Award - Speech |   | hinrichacl@cis.uni-muenchen.de | Lifetime Achievement Award | slideslive.com/38929471 | 38929471
8004 | Lifetime Achievement Award (LAA) - Introduction of the recipient | Hinirich Schütze | hinrichacl@cis.uni-muenchen.de | Lifetime Achievement Award | slideslive.com/38929472 | 38929472
8005 | Distinguished Service Award (DSA) - Introduction of the recipient | Hinirich Schütze | hinrichacl@cis.uni-muenchen.de | Distinguished Service Award | slideslive.com/38929473 | 38929473
8006 | Distinguished Service Award (DSA) - talk by the recipient |   | hinrichacl@cis.uni-muenchen.de | Distinguished Service Award | slideslive.com/38929474 | 38929474
7000 | Announcement/Introduction BP Awards | Joyce Chai, Natalie Schluter, Joel Tetreault | chaijy@umich.edu, natschluter@itu.dk, tetreaul@gmail.com | Best Paper Award | slideslive.com/38929475 | 38929475
9008 | ACL 2023 Call for Bids | Tim Baldwin | tbaldwin@unimelb.edu.au | Conference Announcements | slideslive.com/38929476 | 38929476
